examples/vision/mlpmixer_image_classification.py

```diff
@@ -0,0 +1,275 @@
+"""
+Title: Image Classification with MLP-Mixer
+Author: [Khalid Salama](https://www.linkedin.com/in/khalid-salama-24403144/)
+Date created: 2021/05/30
+Last modified: 2021/05/30
+Description: Implementing the MLP-Mixer model for image classification.
```

Comment:
Please mention the dataset in the description (in general the description should have more info than the title)

Reply:
Done